app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarScrolling.kt

```diff
@@ -18,15 +18,14 @@ package com.duckduckgo.app.browser.omnibar
 
 import android.view.View
 import com.google.android.material.appbar.AppBarLayout
-import com.google.android.material.appbar.AppBarLayout.LayoutParams.SCROLL_FLAG_ENTER_ALWAYS
-import com.google.android.material.appbar.AppBarLayout.LayoutParams.SCROLL_FLAG_SCROLL
+import com.google.android.material.appbar.AppBarLayout.LayoutParams.*
 import javax.inject.Inject
 
 
 class OmnibarScrolling @Inject constructor() {
 
     fun enableOmnibarScrolling(toolbarContainer: View) {
-        updateScrollFlag(SCROLL_FLAG_SCROLL or SCROLL_FLAG_ENTER_ALWAYS, toolbarContainer)
+        updateScrollFlag(SCROLL_FLAG_SCROLL or SCROLL_FLAG_SNAP or SCROLL_FLAG_ENTER_ALWAYS, toolbarContainer)
```

Comment:
this is the new behaviour that fixes https://github.com/duckduckgo/Android/issues/689

Reply:
Thanks for tracking this and linking PR with the issue, really helpful. 👍